code/src/components/MovieCard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { Link } from 'react-router-dom'
+
+export const  MovieCard = ({id, poster_path, title, release_date}) => {
+    return (  
+
+        <section className='movie-card-container'>
+            <Link to={`/movies/${id}`}>
+                <div className='movie-card-overlay'>
+                    <img className='movie-card-image'
+                        src={`https://image.tmdb.org/t/p/w342${poster_path}`}
+                        alt={`${`title`}`}/>
```

Comment:
Just a tip :) The backtick and $ shouldn't be needed if you are not planning to add any other JSX things like in row 12. You can do {title} and it should work.